app/components/SocialsList.tsx

```diff
@@ -32,6 +33,11 @@ const socials = [
 		logo: TwitterLogo,
 		text: "Twitter",
 	},
+	{
+		href: "https://github.com/SquiggleTools/boston-ts-website",
```

Comment:
@JoshuaKGoldberg, do we want to link to `https://github.com/SquiggleTools/boston-ts-website` or `https://github.com/SquiggleTools`?

Reply:
I think the `boston-ts-website`. https://github.com/SquiggleTools will also have SquiggleConf and likely more stuff too.